pycue/opencue/wrappers/layer.py

```
@@ -696,3 +728,36 @@ def stopTime(self, format=None):
         if not format:
             return self.data.stop_time
         return time.strftime(format, time.localtime(self.data.stop_time))
+
+    # pylint: disable=redefined-builtin
+    def startAfter(self, format=None):
```

Ruff (0.16.1)

[error] Function argument `format` is shadowing a Python builtin

(A002)